doc/api/errors.md

```diff
@@ -1835,6 +1835,10 @@ operation.
 
 ### `ERR_INVALID_CALLBACK`
 
+<!-- YAML
+removed: REPLACEME
+-->
+
 A callback function was required but was not been provided to a Node.js API.
```

Comment:
I think we want to remove this section all together and add an entry in `deprecations.md` to list its removal.
```suggestion
```